adal/devApps/AdalUAPTestApp/MainPage.xaml.cs

```diff
@@ -164,5 +161,41 @@ private async void AccessTokenSilentButton_Click(object sender, RoutedEventArgs
                 this.AccessToken.Text = "Auth failed: " + exc.Message;
             }
         }
+
+
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+
+        private async void AcquireTokenIWA_Click(object sender, RoutedEventArgs e) // make sure to use a client id that is configured, such as the one from the .net sample
+        {
+#pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
+
+            this.AccessToken.Text = string.Empty;
+            AuthenticationContext context = new AuthenticationContext("https://login.microsoftonline.com/common");
+            try
+            {
+                AuthenticationResult authResult = await context.AcquireTokenAsync(
+                    "https://graph.microsoft.com",
+                    ClientId, new UserCredential("bogavril@microsoft.com"))
```

Comment:
>"bogavril@microsoft.com") [](start = 49, length = 25)

should this be an input via the UI instead? (perhaps the UI should be grabbing this from the user of the system by default?)